core/src/main/java/org/geysermc/geyser/entity/EntityDefinitions.java

```diff
@@ -279,44 +278,52 @@
             PAINTING = EntityDefinition.inherited((PaintingEntityFactory) PaintingEntity::new, entityBase)
                     .type(EntityType.PAINTING)
                     .build();
-            PRIMED_TNT = EntityDefinition.inherited(TNTEntity::new, entityBase)
-                    .type(EntityType.PRIMED_TNT)
+            TNT = EntityDefinition.inherited(TNTEntity::new, entityBase)
```

Comment:
These are supposed to be in rough alphabetical order, so feel free to move this down.